src/loaders/lottie/tvgLottieBuilder.cpp

```diff
@@ -954,7 +954,7 @@ static void _updateLayer(LottieLayer* root, LottieLayer* layer, float frameNo)
 static void _buildReference(LottieComposition* comp, LottieLayer* layer)
 {
     for (auto asset = comp->assets.data; asset < comp->assets.end(); ++asset) {
-        if (strcmp(layer->refId, (*asset)->name)) continue;
+        if (*asset == nullptr || strcmp(layer->refId, (*asset)->name)) continue;
```

Comment:
So basically, invalid asset shouldn't be in the assets list. Rather than this, I think there is necessary to add an exception handling in parser side. Please check this and make a proper fix.

![image](https://github.com/thorvg/thorvg/assets/3711518/6bc87db8-83c7-4ec8-be34-a102af603420)